letsencrypt/cli.py

```diff
@@ -71,7 +71,8 @@
 
   --apache          Use the Apache plugin for authentication & installation
   --nginx           Use the Nginx plugin for authentication & installation
-  --standalone      Run a standalone HTTPS server (for authentication only)
+  --standalone      Run a standalone webserver for authentication
+
```

Comment:
Nittiest nit: I don't like this space. It breaks up the block of plugin options.